transcom/mymove
# B 20063 int

## [B-20063](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20063)

## Summary
This story adds database migrations to create orders permission flags for client certs. These will be used to differentiate Air Force and Space Force orders permissions. These permissions have been given to all migrations-generated client certs that have Air Force permissions per PO request. The downstream [B-20064](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20064) will follow to update affiliations and issuers accordingly. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Check out the branch and ensure the new database migrations are ran.
2. Access the admin app as any user.
3. Navigate to the "Client Certs" tab on the left menu.
4. Select any client cert, and view the orders permissions.
5. Scroll down until you locate Allow space force orders read / Allow space force orders write.
6. Update the certs and ensure that they save properly.
7. Revert your changes manually for the cert you tested on step 6.
8. Ensure that all certs with Air Force permissions also have Space Force permissions.

### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
<img width="192" alt="image" src="https://github.com/user-attachments/assets/d044af85-5354-446b-b699-f7cc65e271f1">
